Ag/pmu integration

antoniog11 opened the pull request.

antoniog11 committed:
added pmu source files

antoniog11 committed:
added MAX77650 library; Can set output voltages

antoniog11 committed:
Cleaned up pmu init code
- Fixed cli pmu write code
- Added pmu target voltage definitions
- Added printing of target voltages

antoniog11 committed:
Removed JEITA functions

mkimhj commented on cli.c:
```diff
@@ -116,6 +117,95 @@ NRF_CLI_CREATE_STATIC_SUBCMD_SET(m_sub_cam){
+  }
+  else{
+    NRF_LOG_RAW_INFO(NRF_LOG_FLOAT_MARKER " V\n", NRF_LOG_FLOAT(MAX77650_getTV_SBB2() * 0.05 + 2.4));
+  }
```
let's make this a function inside pmu.c, and call into it from the cli.

mkimhj reviewed (commented):
Let's do pmu_init inside powerInit() in main, and also do the set voltages there. 

Can you also add a print message before and after the pmu work so we can understand how long it takes?

print (systemGetTimeMs(): pmu start time %d)
pmu_init
pmu_set_voltages
print (systemGetTimeMs(): pmu end time %d)

something like this ^

antoniog11 committed:
Fixed spi/cli freezing

spi.c/imu.c
- Increased sized of spi buffer to prevent overflow

gpio.h
- Modified pin assignments to match board layout

pmu.c
- Adjusted voltage outputs

imu.c
- Fixed return value for sensor data
